notebooks/BERT_train.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
In line 2 of this notebook, I get the following error: 

Using TensorFlow backend.
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-968c85ba65d6> in <module>
      1 import data_format_utils as dfu
      2 from dataloaders import TrainValDataloader
----> 3 from bert import BertForWSD

/mnt/fs03/home/nlopatina/NLP_word_sense_disambiguation/source/models/bert.py in <module>
      2 import torch
      3 from torch import nn
----> 4 from models.token_layers import TokenClsLayer, SentClsLayer
      5 
      6 

ModuleNotFoundError: No module named 'models'